src/GraphQL/SchemaExtensions.cs

```diff
@@ -205,4 +259,12 @@ public enum ExperimentalIntrospectionFeaturesMode
         /// </summary>
         IntrospectionAndExecution
     }
+
+    [Flags]
+    public enum AutoRegisteringMode
+    {
+        Input = 1,
+        Output = 2,
+        Both = Input | Output
+    }
```

Comment:
Needs xml comments for the enum and each member.

Reply:
Done